vspec/__init__.py

```diff
@@ -308,9 +308,7 @@ def cleanup_flat_entries(flat_model):
 def cleanup_deep_model(deep_model):
     # Traverse the flat list of the parsed specification
     # Is this an include element?
```

Comment:
Does the comment:

`# Is this an include element?`

.. belong to the code you removed? Or also to remaining lines?

Reply:
I wasn't sure what the comment meant, so I kept it. I did not see any relation to includes int the whole function

The comment is from @magnusfeuer  6years, the code below was last touched by @danielwilms. Maybe one of them knows.

Reply:
Removed, I think this comment was an artefact